app/test_main.py

```diff
@@ -2,0 +1,40 @@
+from unittest.mock import patch
+
+import pytest
+
+from app.main import can_access_google_page
+
+
```

Comment:
Try to avoid unnecessary empty lines in your code. They do not add any value and can make the code less readable.